### ISSUE TITLE
[Grid] multiColumn and locked and non-locked and keynavigation upKey

### ISSUE THREAD

**Issue by okinookino** [telerik/kendo-angular#2119]
**Describe the bug**
An error occurs when using MultiColumn and fixed column in grid component and setting KeyNavigation to true. 

**To Reproduce**
First column: kendo-grid-column ([locked] = "true")
Second column: kendo-grid-column ([locked] = "false")
Third column: kendo-grid-column-group

Press the up arrow key on the second row and move to the top

https://angular-vq7aqv.stackblitz.io

**Expected behavior**
I want to make the header row an active cell when I press the up key in the first row cell.

**Browser**
 - OS: Windows7 64bit
 - Browser : Chrome
 - Version: 72.0.3626.121

**Additional context**

Error:
```
zone.js:192 Uncaught TypeError: Cannot read property 'colIndex' of undefined
    at eval (main.js:1)
    at Array.find (<anonymous>)
    at e.offsetRow (main.js:1)
    at e.moveUp (main.js:1)
    at e.onCursorKeydown (main.js:1)
    at e.onKeydown (main.js:1)
    at eval (main.js:1)
    at SafeSubscriber.schedulerFn [as _next] (event_emitter.ts:123)
    at SafeSubscriber.__tryOrUnsub (Subscriber.ts:270)
    at SafeSubscriber.next (Subscriber.ts:212)
```

Code:

```
   @Component({
    selector: 'my-app',
    template: `
        <kendo-grid [data]="gridData" [height]="400" [navigable]="true" >
            <kendo-grid-column field="ProductName" title="Name" [locked]="true" [width]="300" >
            </kendo-grid-column>
            <kendo-grid-column field="ProductName" title="Name2"  [width]="300" [locked]="false">
            </kendo-grid-column>
            <kendo-grid-column-group title="Availability" [width]="600" [locked]="false">
                <kendo-grid-column field="UnitsInStock" title="Units In Stock" [width]="300" [locked]="false">
                </kendo-grid-column>
                <kendo-grid-column field="UnitPrice" title="Unit Price" [width]="300" [locked]="false">
                </kendo-grid-column>
            </kendo-grid-column-group>
        </kendo-grid>
 ```

**Comment by tsvetomir** [comment]
Fixed in the latest [development builds](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by danielkaradachki** [comment]
Fixed in 3.14.0

https://stackblitz.com/edit/angular-vq7aqv-ogj4pg?file=app/app.component.ts
